components/msg/webdriver_msg.rs

```diff
@@ -52,6 +53,14 @@ pub enum WebDriverFrameId {
     Parent
 }
 
+#[derive(Clone, Copy, Deserialize, Serialize)]
+pub struct WebDriverElementRect {
```

Comment:
Great point, didn't see that it impl'd `Serialize`. I'll make the change in the update.